reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -10,6 +10,9 @@
     "AmiId": {
       "Type": "AWS::EC2::Image::Id"
     },
+    "ExistingStack": {
```

Comment:
Since There could be multiple existing stacks, should this be `PrimaryStack` instead?